src/optimizers.py

```diff
@@ -4,7 +4,7 @@
 from losses import losses
 from network import *
 import numpy as np
-
+from numbers import Number
```

Comment:
I will delete this import. 

I've imported it because I was trying to add Error if the value for lrn_rate was not a number. 
But even without this, it raises a ValueError anyway.